src/core/ledger/ledger.js

```diff
@@ -77,6 +77,8 @@ type MutableAccount = {|
   active: boolean,
   // key-value store of blockchain addresses the account receives grain to
   payoutAddresses: PayableAddressStore,
+  // allows for reverse-lookups of IdentityIds
```

Comment:
This comment could be better. What does the array contain? Would be worth noting that it contains the account's main identity id in addition to merged ids, since that isn't fully intuitive.